charger/delta.go

```diff
@@ -141,7 +159,7 @@ func (wb *Delta) Enable(enable bool) error {
 		binary.BigEndian.PutUint16(b, 1)
 	}
 
-	_, err := wb.conn.WriteMultipleRegisters(deltaRegEvseChargingPowerLimit, 1, b)
+	_, err := wb.conn.WriteMultipleRegisters(wb.base+deltaRegEvseChargingPowerLimit, 1, b)
```

Comment:
Sollte hier nicht `deltaRegEvseSuspendCharging` geschrieben werden?

Ich habe allerdings testweise mal versucht den Ladevorgang durch manuelles Setzen des Registers zu pausieren, hat aber keine Auswirkungen gehabt.

Reply:
Ja, eigentlich schon ;)

Reply:
Wenn ich den aktuellen Stand mitlaufen lasse bekomme ich alle paar Minuten eine `charger out of sync` Meldung:

```
[delta ] TRACE 2023/12/23 21:58:54 modbus: send 00 4b 00 00 00 06 01 03 06 42 00 01
[delta ] TRACE 2023/12/23 21:58:54 modbus: recv 00 4b 00 00 00 05 01 03 02 00 00
[lp-1  ] WARN 2023/12/23 21:58:54 charger out of sync: expected disabled, got enabled
[lp-1  ] DEBUG 2023/12/23 21:58:54 charger: guard elapse
[lp-1  ] DEBUG 2023/12/23 21:58:54 guard timer inactive
```

(egal ob `deltaRegEvseChargingPowerLimit` oder `deltaRegEvseSuspendCharging` für das Enable genutzt wird)